docs/releases/1.11.txt

```diff
@@ -739,6 +739,15 @@ Miscellaneous
 
 * Support for IPython < 1.0 is removed from the ``shell`` command.
 
+* ``Widget.build_attrs`` no longer accepts ``kwargs`` parameters.
```

Comment:
I would start off by saying:
```
* The signature of private API method ``Widget.build_attrs()`` changed from
  ``extra_attrs=None, **kwargs`` to ``base_attrs, extra_attrs=None``.
```
Did you test the example code? I tried it by putting it in the `forms.widgets.Input` and running Django's test suite. I get errors like:
```
  File "/home/tim/code/django/django/forms/widgets.py", line 195, in get_context
    'attrs': self.build_attrs(self.attrs, attrs),
TypeError: build_attrs() takes from 1 to 2 positional arguments but 3 were given
```